contrib/lkt/language/lexer.py

```diff
@@ -158,9 +160,10 @@ class Token(LexerToken):
     # Numbers
     (Pattern('[0-9]+'),    Token.Number),
 
-    # Strings
-    (Pattern('{STRING_SQ}|{STRING_DBQ}'), Token.String),
-    (Pattern('[a-zA-Z]{STRING_DBQ}|[a-zA-Z]{STRING_DBQ}'), Token.PString),
+    # Strings & chars
+    (Pattern('{STRING_DBQ}|{STRING_SQ}'),         Token.String),
+    (Pattern('[a-zA-Z]{STRING_DBQ}'), Token.PString),
+    (Pattern('{CHAR_LIT}'),           Token.Char),
```

Comment:
Looks like `'a'` can now be lexed both as a single-quoted string literal, and as a charactel literal. What about we get rid of single-quoted string literals?

Reply:
Yes, thanks for catching that, I actually thought I did get rid of them!